nova/virt/vmwareapi/vm_util.py

```diff
@@ -22,6 +22,7 @@
 import copy
 import functools
 
+from oslo_concurrency import lockutils
```

Comment:
This import seems unused. Seeing that, I know you haven't run `tox -e pep8`. Please update and tell me, when you have run it.